### DISCUSSION TITLE
Option to pass CSS type templates through EE tag parsing

### DISCUSSION THREAD

**Discussion by jHards** [ExpressionEngine/ExpressionEngine#2990] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Simple CSS templates often include a font-family link to font files stored within the EE install. This cannot currently be linked using a path tag because CSS type templates do not parse EE tags.

**Describe the solution you'd like**
A toggle field on the template settings page that will parse any EE tags in the template when selected (default is false).

**Describe alternatives you've considered**
Hard coding the link goes against the idea of using the path variables. Happy to hear any other ideas.

**Teachability, Documentation, Adoption, Migration Strategy**
Would require minor updates to [docs page](https://docs.expressionengine.com/latest/control-panel/template-manager.html).

**Comment by litzinger** [comment]
My personal opinion, and I think many others share this, is to forget that EE even has the option of CSS as a template type. This is a legacy feature that has been around for 10+ years, and there are much, much better ways to manage your CSS files. Use Grunt, Gulp, or whatever to handle your CSS files. Don't use EE to serve CSS files as templates. It's built in template revisions are not comparable to proper version control. Manage all your template files, including CSS, with Git and you'll be better off.

**Comment by derekjones** [comment]
Can you elaborate on CSS templates that link to default control panel EE fonts? In general I agree with @litzinger that front end assets should be managed as static files and not in the CMS, though the option is there when needed. If there's a shortcoming though we would like to address it either with functionality or improved documentation. Thanks!

**Comment by jHards** [comment]
Thanks for the comments gents. Suggest the docs need updating with how it's envisaged this is setup. Even having the CSS template type doesn't seem to align with how you intend people to actually use EE. I for one would be keen to see how you suggest people carry the couple of steps that follow the actual install and go through the recommended security items e.g.

- Create a homepage by with embedded templates for the header and footer
- Upload CSS (from whatever source) and link to the template
- Upload and include JS

Not sure some of the suggestions in the docs are still up to date with your current thinking.

**Comment by robsonsobral** [comment]
I would like to remember one reason to keep CSS as a template, specially on CDN cases: cache busting.
